task4/src/services/groupService.js

```diff
@@ -0,0 +1,71 @@
+export default class GroupService {
+    constructor(models) {
+        this.model = models.Group;
+        this.userModel = models.User;
+    }
+
+    getAllGroups() {
+        try {
+            const data = this.model.findAll({
```

Comment:
```
return this.model.findAll({
                include: [
                    {
                        model: this.userModel,
                        as: 'users',
                        required: false,
                        attributes: ['id', 'login'],
                        through: { attributes: [] }
                    }
                ]
            })

No need to wrap in try catch here,  a promise can be returned

Reply:
So, am I understand correctly, that we can rely on all synchronous model's methods, and we can just return promise for all services methods ?